test/Swashbuckle.AspNetCore.SwaggerGen.Test/Fixtures/Fakes/FakeApiDescriptionGroupCollectionProvider.cs

```diff
@@ -73,7 +78,12 @@ public ApiDescriptionGroupCollection ApiDescriptionGroups
         {
             var descriptor = new ControllerActionDescriptor();
 
+#if NETCOREAPP3_0
+            // SEE : https://github.com/aspnet/AspNetCore/issues/14454#issuecomment-535571938
+            descriptor.SetProperty(new ApiDescription());
```

Comment:
You might need to use reflection to do this until an equivalent API is added:

```C#
var type = typeof(ControllerBase).Assembly.GetType("Microsoft.AspNetCore.Mvc.ApiDescriptionActionData", throwOnError: true);
var instance = Activator.CreateInstance(type);
actionDescriptor.Properties[type] = instance;
```

Reply:
oO i think it would have took me few days/week to realize that
naming got me ;) that will teach me to reach/check what i use ahahah

thx a lot @pranavkm

Reply:
> You might need to use reflection to do this until an equivalent API is added:

Is this something ASP.NET Core is just missing?

Reply:
@pranavkm it fixed all test except one, i'll take a look at it today, in any case if you got another idea on what i missed

Reply:
@davidfowl i'm not really sure to be honest, if you check this commit :
https://github.com/tebeco/Swashbuckle.AspNetCore/commit/ed7888be8fd41141279583916afbc42bc9d9d6d8
This is a `Fake` for `IApiDescriptionGroupCollectionProvider`

The fix proposed by @pranavkm  seems to have fixed 46 `IntegrationTest` with the existing codebase.
Is there a better code fix ? that would be "doable" with the existing subset of API in 3.0 ?
(Also note that as of today the repo is not using either `UseRouting()` or `UseEndpoints()` (just trying to fully cross target in this PR)

The main issue is this :
https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/ad91d72078e1e2276824b38e2483c1d8ab6e5376/src/Swashbuckle.AspNetCore.Swagger/Swashbuckle.AspNetCore.Swagger.csproj#L21-L24

(i'm not sure how "sure" it could be to track `Microsoft.AspNetCore.Mvc.Core` (and other package from 2.0.0) in 3.0 app, so i attempted the full cross-targeting